definitions/horizontal_scaling.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Horizontal Scaling
+status: Feedback Appreciated
+category: Concept
+---
+
+## Horizontal Scaling
+
+### What it is
+
+In [cloud computing](https://github.com/cncf/glossary/blob/main/definitions/cloud_computing.md) perspective Horizontal Scaling is type of [scalability](https://github.com/cncf/glossary/blob/main/definitions/scalability.md) whose approach is to add additional instances instead of moving into a larger instance size.
+
+For example, let us consider we have a system of 4GB RAM and we want to increase it's capacity to 16GB RAM, Horizontal Scaling means doing this by making a system of 4 X 4GB RAM rather than switching to a 16GB RAM system.
+
+It is an approach to enhance the performance of the server [node](https://github.com/cncf/glossary/blob/main/definitions/nodes.md) by adding new instances of the server to already existing servers in order to distribute the workload equally. In simple words it aims to decrease the server's load rather than expanding capacity of the individual server.
+
+Load balancing, clustering, and distributed file systems are some of the crucial and popular strategies of horizontal scaling. It caters to portioning of data where each [node]((https://github.com/cncf/glossary/blob/main/definitions/nodes.md)) contains single part of the data.
+
+### Problem it addresses
+
+With growing demands of the application the system should be able to scale by increasing/growing/managing the resources for the customers using it. The latency of the API calls might increase due to heavy traffic, this is the problem that can be solved by horizontal scaling.
+
+### How it helps
+
+Horizontal scaling helps as it can provide limitless scaling with endless additions to server instances. It enhances the I/O concurrency, and also reduces the load on existing [nodes](https://github.com/cncf/glossary/blob/main/definitions/nodes.md) and expands disk capacity.
```

Comment:
So at a minimun we don't want to use any abbreviations so you'd need to spell out I/O. Even better consider that the audience for this is folks that don't necessarily know a ton of computer science terms and instead use language that conveys your meaning in a more accessible way. Example:

Horizontal scaling allows an application to process a greater number of requests by creating more nodes, or application instances, that can perform the work. If a single node can handle 1000 requests per second then each additional node should increase the total number of requests by around 1000 requests per second.

Reply:
I get it that it is folks who would read to understand. From next time I would try to keep it in a way that anyone would could understand and as simple as possible

Reply:
Awesome! Do you want to make the changes?

Reply:
@JasonMorgan Ofcourse why not